sql/join_boroughs.sql

```diff
@@ -0,0 +1,29 @@
+SELECT
+    a.*,
+    b.borocode::VARCHAR AS borough
+INTO _combined
+FROM combined a 
+LEFT JOIN dcp_boroboundaries b ON ST_intersects(a.geom, b.wkb_geometry);
+
+DROP TABLE combined;
+SELECT * INTO combined FROM _combined;
+DROP TABLE _combined;
+VACUUM ANALYZE combined;
```

Comment:
I don't love this select into a new table and dump right back, but given that we're joining and getting new rows in the join it seemed like the simplest way to go. Open to suggestions here